server/src/main/scala/Server.scala

```diff
@@ -0,0 +1,17 @@
+package server
+
+import akka.stream.ActorMaterializer
+import akka.actor.ActorSystem
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/olivier-blanvillain/19x19-net/file/2594820883/issues/source?bid=3301982&fileBranchId=3306761#l4)